src/main/webapp/style.css

```diff
@@ -50,6 +50,24 @@ body {
   display: none;
 }
 
+.task {
+  display: flex;
+}
+
+.task-title {
+  flex-grow: 1;
+}
+
+.task-duration {
+  align-self: flex-end;
+  margin-right: 10px;
+  min-width: 122px;
```

Comment:
This is a good place to add a comment and explain why this has to be exactly `122px`, otherwise somebody might change it without knowing its exact purpose. (and if there is no specific reason, maybe it's not necessary?)